files/matreshka/update.json

```diff
@@ -3,6 +3,13 @@
   "name": "matreshka",
   "repo": "matreshkajs/matreshka",
   "files": {
-    "include": ["matreshka.min.js", "matreshka.js", "matreshka.min.map"]
+    "include": [
+		"matreshka.min.js",
+		"matreshka.js",
+		"matreshka.min.map",
+		"magic/matreshka-magic.min.js",
```

Comment:
To make sure. Do you want the `magic/` part to remain and the bot to create that folder or you want all the files in the same directory?